frontend/src/shared/nameHelper.ts

```diff
@@ -70,3 +70,20 @@ export function osName(os?: Ios) {
 export function safeFilename(name: string) {
   return name.replace(REGEX_NOT_FILE_SAFE, '-')
 }
+
+export function serviceNameValidation(name: string, validateLength?: boolean) {
+  const value = name.replace(REGEX_NAME_SAFE, '')
+  if (value !== name) {
+    return  { 
+      error: 'Device names can only contain alpha numeric characters.',
```

Comment:
Sorry missed this reference too:
```suggestion
      error: 'Can only contain alpha numeric characters.',
```